src/views/templates/details/tabs/disks/components/DiskRowActions.tsx

```diff
@@ -62,12 +62,13 @@ const DiskRowActions: React.FC<DiskRowActionsProps> = ({ diskName, vm, onUpdate,
         isOpen={isOpen}
         obj={vm}
         onSubmit={onDelete}
-        headerText={t('Detach {{diskName}} disk', { diskName })}
+        headerText={t('Detach {{diskName}} disk?', { diskName })}
         submitBtnText={deleteBtnText}
         submitBtnVariant={ButtonVariant.danger}
       >
         <Trans t={t}>
-          Are you sure you want to detach <strong>{diskName} </strong>
+          Are you sure you want to detach <strong>{diskName} </strong>in namespace{' '}
```

Comment:
Why not use `DeleteResourceMessage` here?

Reply:
Because here we use the verb "detach", not "delete" as it is in `DeleteResourceMessage`, so I better left this as it is.